Composer/packages/server/src/models/asset/assetManager.ts

```diff
@@ -159,9 +160,10 @@ export class AssetManager {
     dstDir: string,
     projectName: string
   ): Promise<boolean> {
+    log('About to instantiate a template!', dstDir, generatorName, projectName);
     this.yeomanEnv.cwd = dstDir;
```

Comment:
Ah I see what happened, this.yeomenEnv.cwd was not changing the process cwd as expected hence your addition of proecces.chdir. 

Does that mean we no longer need this.yeomenEnv.cwd  as its not really doing anything?